### PR TITLE
ovn: actually apply hw-offload setting

### PR DESCRIPTION
**Pull request by booxter**
The code was constructing a new dict but never applying it.

### CODE REVIEW

File: roles/edpm_ovn/tasks/configure.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@
 
 - name: Configure OVS other_config
   ansible.builtin.shell: >
-    ovs-vsctl set open . {% for key, value in edpm_ovn_ovs_other_config.items() %} other_config:{{ key }}={{ value }} {% endfor %}
+    ovs-vsctl set open . {% for key, value in other_config.items() %} other_config:{{ key }}={{ value }} {% endfor %}
```
Comment (karelyatin) [inline roles/edpm_ovn/tasks/configure.yml:69]: need to change L61 instead so it works with edpm_enable_hw_offload=false:-
edpm_ovn_ovs_other_config: "{{ edpm_ovn_ovs_other_config | combine(hw_offload) }}"

**Comment by booxter** [comment]
@jpodivin would you mind confirming this is the right approach to avoid duplicating code over multiple molecule scenarios? (using symlinks + common tasks yaml that is included in multiple places).

File: roles/edpm_ovn/molecule/default/verify-tasks.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    - name: Check if nftables rule exists in /etc/nftables/edpm-rules.nft
+      become: true
+      ansible.builtin.shell: grep -q "{{ item }}" /etc/nftables/edpm-rules.nft
+      register: rule_exists
+      ignore_errors: true
+      loop:
+        - "118 neutron vxlan networks"
+        - "119 neutron geneve networks"
+        - "120 neutron geneve networks no conntrack"
+        - "121 neutron geneve networks no conntrack"
```
Comment (slawqo) [inline roles/edpm_ovn/molecule/default/verify-tasks.yml:39]: I wonder if wouldn't it be better to move to testinfra as verifier. It has easy to used dedicated modules to e.g. test if file exists and if it contains some strings, see https://testinfra.readthedocs.io/en/latest/modules.html#testinfra.modules.file.File
I'm using it now in the edpm_neutron_{metadata,dhcp} roles to test sidecars

Comment (booxter) [reply roles/edpm_ovn/molecule/default/verify-tasks.yml:39]: Perhaps though here I'm just moving code around.

**Comment by booxter** [comment]
recheck

**Comment by booxter** [comment]
Failed to download some files for the env, unrelated failure

**Comment by booxter** [comment]
recheck

**Review by jpodivin — APPROVED** [review]
Great work!

**Pull request merged**